docs/GraphQL/generatingApiKey.rst

```diff
@@ -1,4 +1,14 @@
 Generating new API keys
 =============================
 
-TODO
\ No newline at end of file
+To generate a new API key navigate to the User settings page. You can find it by clicking your user in the top right corner of the page. When you are in the user setup page navigate to the API keys tab. 
+
+To create a new API key click the *create new API key* button then go ahead, give it a descriptive name and hit save. You should now be forwarded to your new API key. Please keep your new API key a secret as it gives whomever has the key access to read and manipulate your data on the platform.
+
```

Comment:
Add "that" so that the sentence is: 
"as it gives whomever that has the key access to read and manipulate your data on the platform."